app/controllers/LoginController.java

```diff
@@ -0,0 +1,156 @@
+package controllers;
+
+import javax.inject.Inject;
+
+import play.mvc.Result;
+import play.data.FormFactory;
+import play.data.Form;
+import views.html.login;
+import views.formdata.UserFormData;
+import views.formdata.LoginFormData;
+
+import java.sql.Connection;
+import java.sql.PreparedStatement;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import play.db.Database;
+
+import codeu.chat.client.Controller;
+import codeu.chat.client.View;
+import codeu.chat.util.RemoteAddress;
+import codeu.chat.util.connections.ClientConnectionSource;
+import codeu.chat.util.connections.ConnectionSource;
+import codeu.chat.util.Logger;
+import codeu.chat.client.ClientContext;
+
+/**
+ * Created by HuyNguyen on 4/4/17.
+ */
+public class LoginController extends play.mvc.Controller {
+
+  private final static Logger.Log LOG = Logger.newLog(LoginController.class);
+  private ClientContext clientContext;
```

Comment:
Is there a way to make this a session variable? As it stands, it looks like multiple people can't use the web server at the same time.